click/utils.py

```diff
@@ -414,3 +414,17 @@ def get_app_dir(app_name, roaming=True, force_posix=False):
     return os.path.join(
         os.environ.get('XDG_CONFIG_HOME', os.path.expanduser('~/.config')),
         _posixify(app_name))
+
+
+class PacifyFlushWrapper:
```

Comment:
Please add a doc string describing what this class is used for and why.

Since we're still on Python 2 & 3, have it inherit from 'object' to be a new style class as well.